src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1285,6 +1285,70 @@ featurizers:
     # we introduce the tabular data type itself).
     status: External
 
+  # ----------------------------------------------------------------------
+  - name: LagLeadOperatorFeaturizer
+    estimator_name: LagLeadOperatorEstimator
+    release_version: 0.4.0
+    num_output_columns: 1
+    description: |-
+      Computes lags and leads for values. Works for general time series data sets.
+      For sparse time series data sets, forward fill imputer will be applied.
+
+      C++-style pseudo signature:
+        template <typename T> double execute(std::tuple<std::chrono::system_clock::time_point, T, uint8> const &value);

```

Comment:
Change the uint8 to a date to match the time column

Reply:
Also, we don't currently support tuples in the code generation, so run this on your branch first with status as pending to make sure nothing is broken for now until we can add in tuples.